lib/client/backend_client.dart

```diff
@@ -20,32 +25,56 @@ class BackendClient {
     } catch (e) {
       print('Error during HTTPrequest $e');
       return 'Error during HTTP request: $e';
-    }
+    } // } finally {
+    //   // JACKSON: The finally block was added due to adding the httpClient rather than the one-off requests
+    //   httpClient.close();
+    // }
   }
 
   Future<dynamic> postData({required String uri, dynamic body}) async {
     try {
       final Uri url = Uri.http(localhost, uri);
-      final response = await http.post(
+      final response = await httpClient.post(
         url,
         headers: headers,
         body: jsonEncode(body),
       );
-      if (response.statusCode == HttpStatus.created) {
+      if (response.statusCode == 201) {
```

Comment:
I didn't know this enum was there before. I like the enum better. It's giving context to the int literal. `HttpStatus.created` = 201

Reply:
done